doc/2/api/controllers/security/upsert-user/index.md

```diff
@@ -0,0 +1,126 @@
+---
+code: true
+type: page
+title: upsertUser
+---
+
+# upsertUser
```

Comment:
You need to add the `<SinceBadge version="auto-version"/>` element to inform the user that this action is available since the last release. the `auto-version` field will be automatically changed during release.